src/components/Home.js

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { getData } from '../redux/home/home';
+import Card from './Card';
+
+export default function Home() {
```

Comment:
- [ ] Kindly, you should follow these [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), including:
- Typography: font face, size and weight.
- Layout: composition and space between elements. Kindly improve your design to match the original design and don't forget to add the missing `icon`  to the home page and use the appropriate icon to the details page


||  Original [design guidelines](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs))  ||
|-|-----------------------------------|-|
||![go1](https://user-images.githubusercontent.com/57562869/157501449-50bab51c-5494-447a-a950-0c1abbba504d.png)||

||    `Home` page      ||      `Detail` page        ||
|-|----------------------|-| ----------------------|-|
||![00000000000000](https://user-images.githubusercontent.com/57562869/159724041-140efb44-ffde-490c-bcea-e7a3e94b1c2b.png)||![00000222222](https://user-images.githubusercontent.com/57562869/159724124-f0a6b2a2-59bd-4fa4-b98e-91b0f4a3040c.png)||